Blockzilla/BrowserViewController.swift

```diff
@@ -364,6 +365,37 @@ class BrowserViewController: UIViewController {
 
         Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.click, object: TelemetryEventObject.eraseButton)
     }
+    
+    private func requestReviewIfNecessary() {
+        let currentLaunchCount = UserDefaults.standard.integer(forKey: UIConstants.strings.userDefaultsLaunchCountKey)
+        let threshold = UserDefaults.standard.integer(forKey: UIConstants.strings.userDefaultsLaunchThresholdKey)
+
+        if threshold == 0 {
+            UserDefaults.standard.set(14, forKey: UIConstants.strings.userDefaultsLaunchThresholdKey)
+        }
+
+        // Make sure the request isn't within 90 days of last request
+        let minimumDaysBetweenReviewRequest = 90.0
+        let currentTime = Double(Date().timeIntervalSince1970)
+        let previousRequest = UserDefaults.standard.double(forKey: UIConstants.strings.userDefaultsLastReviewRequestDate)
```

Comment:
How about UserDefaults `setObject` and store a Date() object. Numeric timestamps are tricky and best avoided (ambiguity in: units and the t-zero).